charts/agent-control-deployment/templates/namespace-agents.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $subAgentsNamespace := index (include "newrelic-agent-control.config.content" . | fromYaml) "k8s" "namespace_agents" -}}
```

Comment:
I am not following why you are not simply getting it from .Values.subAgentsNamespace